pkg/sshd/sshd.go

```diff
@@ -31,7 +30,7 @@ const (
 func ParseHostKeys(c cookoo.Context, p *cookoo.Params) (interface{}, cookoo.Interrupt) {
```

Comment:
IIRC it verifies that the keys are indeed proper SSH host keys. Other than that I think this code is useless. Want me to remove it?